clients/admin-ui/src/features/config-wizard/ConfigWizardWalkthrough.tsx

```diff
@@ -85,23 +84,20 @@ const ConfigWizardWalkthrough = () => {
                 {reviewStep === 1 && (
                   <DescribeSystemStep
                     system={system}
-                    onCancel={handleCancelSetup}
                     onSuccess={handleSuccess}
                     abridged
                   />
                 )}
                 {reviewStep === 2 && system && (
                   <PrivacyDeclarationStep
                     system={system}
-                    onCancel={handleCancelSetup}
                     onSuccess={handleSuccess}
                     abridged
                   />
                 )}
                 {reviewStep === 3 && system && (
                   <ReviewSystemStep
                     system={system}
-                    onCancel={handleCancelSetup}
                     onSuccess={() => dispatch(changeReviewStep())}
```

Comment:
I would suggest adding the `dispatch(changeReviewStep)` to a new event handler to be consistent with the other event handlers. Just to be consistent with your other event handler declarations.